crates/net/downloaders/src/file_client.rs

```diff
@@ -18,6 +18,11 @@ use tokio_stream::StreamExt;
 use tokio_util::codec::FramedRead;
 use tracing::{trace, warn};
 
+/// Byte length of chunk to read from chain file. Equivalent to one static file, full of max
+/// size blocks (500k headers = one static file, transactions from 500k blocks = one static
+/// file).
+const BYTE_LEN_CHUNK_CHAIN_FILE: u64 = 500_000 * 12 * 1_000_000;
```

Comment:
maybe we can increase it to 1GB? 

Also, I'm unsure on where the equivalency is here. Static files are of variable sizes, I think we can remove this part of the comment

Reply:
one static file, **full of max size blocks**

added it to make the number have meaning to new comers

Reply:
changed it to 1 gb